### ISSUE TITLE
value of type 'UIStackView' has no member 'snapToSuperview'

### ISSUE THREAD

**Issue by lexuanquynh** [zzmasoud/ZZImageSlider#2]
Looks like you use snapkit, right?
<img width="811" alt="Screenshot 2023-10-06 at 07 12 12" src="https://github.com/zzmasoud/ZZImageSlider/assets/17588791/ff5f7ced-3e0f-49dd-97bc-8359ab3fb64a">

**Comment by zzmasoud** [comment]
@lexuanquynh yes, actually I'm moving the project to SwiftUI and it may takes a little while! sorry about that. (it's on the develop branch)

**Comment by webbrain-one** [comment]
**Comment:**  

Thanks for catching this! 😊 A pull request ([PR #3](https://github.com/zzmasoud/ZZImageSlider/pull/3)) has been opened to resolve the compiler error where `UIStackView` lacks the `snapToSuperview` member. The fix adjusts the SnapKit usage to properly configure the layout constraints, ensuring compatibility and error-free compilation.  

Appreciate your report and the contributor’s quick fix! 🚀
